test/database.ts

```diff
@@ -120,6 +120,37 @@ class FakeSessionPool extends EventEmitter {
   release() {}
 }
 
+export class FakeMultiplexedSession extends EventEmitter {
+  calledWith_: IArguments;
+  constructor() {
+    super();
+    this.calledWith_ = arguments;
+  }
+  createSession() {}
+  getSession() {}
+}
+
+export class FakeSessionFactory extends EventEmitter {
+  calledWith_: IArguments;
+  constructor() {
+    super();
+    this.calledWith_ = arguments;
+  }
+  getSession(): FakeSession | FakeMultiplexedSession {
+    if (process.env.GOOGLE_CLOUD_SPANNER_MULTIPLEXED_SESSIONS) {
```

Comment:
This condition should be reversed ?

Reply:
yeah, thanks for pointing this out
updated